src/encoded/static/components/matrix_mouse_development.js

```diff
@@ -129,8 +122,10 @@ const analyzeSubCategoryData = (subCategoryData, columnCategory, colMap, colCoun
 // In the future there are likely to be additions to the data which will require updates to this function
 // For instance, ages measured by months will likely be added
 function sortMouseArray(a, b) {
-    const aStage = a.split(' (')[0];
-    const bStage = b.split(' (')[0];
+    const aStage = a.split(/ (.+)/)[0].replace('embryonic', 'embryo');
```

Comment:
I actually think this will work no matter what. It's the sorting function for `mouseAgeFullArray` which is the array of bucket keys (life_stage_age properties). If one of those happened to be the empty string, then a or b would be an empty string and aAge/aStage or bAge/bStage would be empty strings and they'd just get sorted as such.